doc/sphinxman/document_tests.pl

```diff
@@ -60,6 +60,15 @@
    "gdma/"      => "gdma",
    "dkh/"       => "dkh",
    "erd/"       => "erd",
+   "v2rdm_casscf/" => "v2rdm_casscf",
+   "snsmp2/"    => "snsmp2",
+   "simint/"    => "simint",
+   "gpu_dfcc/"  => "gpu_dfcc",
+   "gcp/"       => "gcp",
+   "cookbook/"  => "cookbook",
+   "json/"      => "json",
+   "psi4numpy/" => "psi4numpy",
+   "python/"    => "python",
 );
```

Comment:
didn't realize the good ol' perl script is still handling this.  blast from the past!